src/constants.ts

```diff
@@ -0,0 +1,133 @@
+export enum EventType {
```

Comment:
Good call on using singular versions of the words. TIL - https://stackoverflow.com/questions/1335959/singular-or-plural-for-enumerations?lq=1